profiles/models.py

```diff
@@ -176,9 +180,16 @@ class Profile(models.Model):
     updated_on = models.DateTimeField(blank=True, null=True, auto_now=True)
 
     @transaction.atomic
-    def save(self, *args, **kwargs):
-        """Set the student_id number to the PK number"""
-        # first save the profile
+    def save(self, *args, update_thumbnails=False, **kwargs):
```

Comment:
If I have a nitpick, I would rather the `update_thumbnails` kwarg be named `update_image` since we want to update the thumbnails any time the actual profile image is updated. Updating the thumbnails is an implementation detail that is handled by the profile `save` method. You can take or leave this one